src/orquestra/sdk/_shared/__init__.py

```diff
@@ -0,0 +1,34 @@
+from schema import result_is_packed
+
+from ._graphs import iter_invocations_topologically
+from ._logs import (
+    LogAccumulator,
+    LogOutput,
+    LogReader,
+    LogStreamType,
+    WorkflowLogs,
+    is_env_setup,
+    is_worker,
+)
+from ._regex import SEMVER_REGEX, VERSION_REGEX
+from ._spaces import Project, ProjectRef, Workspace
+from ._storage import BaseModel, TypeAdapter
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick (non-blocking):** ‏I would rather not export these Pydantic backwards compat classes from the root.

Reply:
Oh, good point! Any suggestions for the module name?

Reply:
Maybe the "orqdantic" package? WDYT? :D